### ISSUE TITLE
Fixed tooltip takes effect only the second time. 

### ISSUE THREAD

**Issue by BoyWithSilverWings** [PygmySlowLoris/vue-fab#14]
### Expectation
Fixed tooltip should produce a `fixed` tooltip shown on hover

### What actually happens
No tooltip is shown first time. From the second time, tooltip is shown. 

### Steps to reproduce. 
1. Set initial value of `:fixed-tooltip` to true.
![ezgif-5-29634b857b](https://user-images.githubusercontent.com/8883368/35773989-5bc50c7a-0988-11e8-8da9-e94f0c0919db.gif)

**Comment by BoyWithSilverWings** [comment]
It seems to be because of [this function.](https://github.com/PygmySlowLoris/vue-fab/blob/master/src/FAB.vue#L264) 
```
showTooltip(show) {
   if (show && this.$refs.actions && this.fixedTooltip) {
         setTimeout(() => {
            this.$refs.actions.forEach((item) => {
                  if(this.toggle) {
                     item._tooltip.show();
                  }
             });
         },700);
    }
}
```
`this.$refs.actions` is `undefined until it is actually triggered for the first time. This makes it an `[]`
Why is this check in place?

**Comment by Edujugon** [comment]
It's resolved in your PR #15
